src/test/java/com/radixdlt/serialization/ECDSASignaturesSerializationTest.java

```diff
@@ -0,0 +1,45 @@
+package com.radixdlt.serialization;
+
+import com.google.common.collect.ImmutableMap;
+import com.radixdlt.TestSetupUtils;
+import com.radixdlt.crypto.CryptoException;
+import com.radixdlt.crypto.ECDSASignature;
+import com.radixdlt.crypto.ECDSASignatures;
+import com.radixdlt.crypto.ECKeyPair;
+import org.junit.BeforeClass;
+
+import java.math.BigInteger;
+
+
+/**
+ * Serialization for Signatures to JSON.
+ */
+public class ECDSASignaturesSerializationTest extends SerializeObjectEngine<ECDSASignatures> {
```

Comment:
this test fails right now, due to serialization issues with (Immutable)`Map<ECPublicKey, Signature>`